script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5_1.txt

```diff
@@ -83,12 +83,12 @@ Yes, I really do understand.
 I was lusting after her. I was excited to see this girl.
 <Page20>
 It's the same with my eyes.
-I already understood that these 'lines' would cut things like cutting paper.
+I already understood that these 'lines' would cut things like paper.

```

Comment:
Maybe "that these 'lines' would **let** me cut [through] things like paper", since otherwise it feels like the lines are what's doing the cutting
> あの『線』が紙を切るようにモノを切断してしまうと分かっていたのなら。